### PR TITLE
added links to the practitioner guide videos

### PR DESCRIPTION
**Pull request by geekygirldawn**

### CODE REVIEW

File: practitioner-guides/introduction.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ You should also continue to monitor it over the long-term to see if your improve
 # Additional Reading
 
 * [Practitioner Guide series](https://chaoss.community/about-chaoss-practitioner-guides/) with guides to help you improve responsiveness, contributor sustainability, organizational participation, and more.
+* A short video about each guide can be found in the [Practitioner Guides Playlist](https://www.youtube.com/watch?v=xTK2xsd4jZE&list=PL60k37cxI-HSHV4-rEsWMzExw2y2Oq79Z) in the CHAOSS YouTube channel.
```
Comment (ElizabethN) [inline practitioner-guides/introduction.md:62]: I think this should be https://www.youtube.com/playlist?list=PL60k37cxI-HSHV4-rEsWMzExw2y2Oq79Z

File: practitioner-guides/organizational-participation.md [hunk 2]
```diff
@@ -94,6 +94,7 @@ It can take time for projects to recruit users and contributors, so it is partic
 
 # Additional Reading
 
+* We have a [short video](https://www.youtube.com/watch?v=xTK2xsd4jZE&list=PL60k37cxI-HSHV4-rEsWMzExw2y2Oq79Z&index=4) (< 3 minutes) devoted to this guide on the CHAOSS YouTube channel.
```
Comment (ElizabethN) [inline practitioner-guides/organizational-participation.md:97]: I think this link should be this: https://www.youtube.com/watch?v=mztEQMgam5Y&list=PL60k37cxI-HSHV4-rEsWMzExw2y2Oq79Z&index=4

File: practitioner-guides/website-landing.md [hunk 3]
```diff
@@ -9,4 +9,6 @@ These guides will be especially useful for Open Source Program Offices (OSPOs),
 * [Practitioner Guide: Responsiveness](https://chaoss.community/practitioner-guide-responsiveness/)
 * [Practitioner Guide: Organizational Participation](https://chaoss.community/practitioner-guide-organizational-participation/)
 
+A short video about each guide can be found in the [Practitioner Guides Playlist](https://www.youtube.com/watch?v=xTK2xsd4jZE&list=PL60k37cxI-HSHV4-rEsWMzExw2y2Oq79Z) in the CHAOSS YouTube channel.
```
Comment (ElizabethN) [inline practitioner-guides/website-landing.md:12]: I think this link should be this: https://www.youtube.com/playlist?list=PL60k37cxI-HSHV4-rEsWMzExw2y2Oq79Z

**Comment by geekygirldawn** [comment]
@ElizabethN wow! I'm not sure how I mangled those links so badly! Can you do a quick double check and merge it when you get a chance?

**Review by ElizabethN — APPROVED** [review]

**Comment by ElizabethN** [comment]
Looks great @geekygirldawn! ⭐

**Pull request merged**